bot.py

```diff
@@ -847,6 +851,51 @@ def check(m):
             else:
                 await reply(message, "Could not find player " + parameters)
 
+@cmd('choose', [2, 0], "```\n{0}choose <player1> and <player2>\n\nIf you are a matchmaker, Selects two players to fall in love. You may select yourself as one of the lovers.```")
+async def cmd_choose(message, parameters):
+    if not session[0] or get_role(message.author.id, 'role') not in COMMANDS_FOR_ROLE['choose'] or not session[1][message.author.id][0] or not message.channel.is_private:
+        return
+    if parameters == "":
+        await reply(message, roles[session[1][message.author.id][1]][2])
+    else:
+        if get_role(message.author.id, 'role') == 'matchmaker' and 'match' in session[1][message.author.id][4] and not session[2]:
+            parameters = parameters.lower().split(" and ")
+            if len(parameters) == 2:
+                player1 = get_player(parameters[0])
+                player2 = get_player(parameters[1])
```

Comment:
Add an error message if two players were not chosen? (currently it just returns if the command `match 1` is used)